src/models/SessionLogoutInterval.ts

```diff
@@ -0,0 +1,19 @@
+import { INTERVAL_NAMES } from '../constants';
+
+export class SessionLogoutInterval {
+  public interval: number;
+  public name: string;
+
+  constructor(interval: number, name: string) {
+    this.interval = interval;
+    this.name = name;
+  }
+}
+
+export const sliArray = [
```

Comment:
nit: SLI_ARRAY

Reply:
this shouldn't belong in a model class. model classes should just be a representation of an instance and what that instance can do.

move this array to the `SessionStore` (or better yet to a new `SettingsStore`).